### ISSUE TITLE
Website builds should hit preload servers

### ISSUE THREAD

**Issue by lidel** [ipfs/testing#147]
## Which Project is this about?

All websites built via [website()](https://github.com/ipfs/jenkins-libs/blob/master/vars/website.groovy) script from ipfs/jenkins-libs

##  Improvement

Right now websites are only pinned on jenkins master, which makes initial seeding suboptimal.
I know there is work is in progress to have it pinned on the pinbot cluster, however until that happens we could just manually connect to preload nodes and add a simple preload step around [website.groovy#L116](https://github.com/ipfs/jenkins-libs/blob/2e5f6ae98bcd6fbcca8825f507435283a9bd9da1/vars/website.groovy#L116) and [#L144](https://github.com/ipfs/jenkins-libs/blob/2e5f6ae98bcd6fbcca8825f507435283a9bd9da1/vars/website.groovy#L144) to improve the number of initial seeds:

```console 
$ curl -s https://node{0|1}.preload.ipfs.io/api/v0/refs?r=true&arg={websiteHash}
```

cc @VictorBjelkholm

**Comment by VictorBjelkholm** [comment]
Agree! Something that would help and be quick to implement until we have ipfs-cluster pinning.

We should only run the preload command from the master to not take up more linux workers, and the example you gave there seems perfect!

PRs welcome! :)

**Comment by lidel** [comment]
PR it is: https://github.com/ipfs/jenkins-libs/pull/37 :)
